### PR TITLE
QE Test : rt app with no exec probes

### PR DESCRIPTION
**Pull request by bnshr**

### CODE REVIEW

File: tests/performance/helper/helper.go [hunk 1]
```diff
@@ -67,3 +116,70 @@ func RedefinePodWithSharedContainer(pod *corev1.Pod, containerIndex int) {
...
+	return outStr, errStr, err
+}
+
+// ExecCommand runs command in the pod and returns buffer output.
+func ExecCommandContainer(
+	pod *corev1.Pod, command string) (stdout, stderr string, err error) {
+	commandStr := []string{"sh", "-c", command}
+
+	var buffOut bytes.Buffer
+
```
Comment (sebrandon1) [inline tests/performance/helper/helper.go:135]: No need for the extra space.

File: tests/performance/helper/helper.go [hunk 2]
```diff
@@ -51,6 +60,46 @@ func DefineExclusivePod(podName string, namespace string, image string, label ma
 	}
 }
 
+func DefineRtPod(podName string, namespace string, image string, label map[string]string) *corev1.Pod {
```
Comment (sebrandon1) [inline tests/performance/helper/helper.go:63]:
```suggestion
func DefineRtPod(podName, namespace, image string, label map[string]string) *corev1.Pod {
```

**Review by sebrandon1 — APPROVED** [review]
Some minor things noted.  Approved!

File: tests/performance/helper/helper.go (hunk 1 above)
Comment (bnshr) [reply tests/performance/helper/helper.go:135]: These are added because go-linter failed in the CI pipeline.

**Review by theemj — APPROVED** [review]
looks good to me!

**Pull request merged**